src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: $transition-time;
```

Comment:
It's more correct to specify for which rule the transition time is set, for example: `transition: color $transition-time`